examples/source/advertising-analytics/Tracking_Core_Web_Vitals.html

```diff
@@ -99,3 +99,48 @@
       }
     </script>
   </amp-analytics>
+
+
+  <!--
+  ## Comparing performance on Cache vs Origin
+  -->
+  <!--
+    AMP Caches can improve your code with a number of optimizations, but it is important to not rely on a Cache for your site's performance. A good way to make sure you are delivering a great experience for all your users is to keep track of how your pages' Core Web Vitals are performing on and off of the Cache.
```

Comment:
Few notes on this section:
* AMP Caches provide more than just code optimisations. It's probably not necessary here to list all, but if you list only one it might be confusing. Maybe stay more generic here 
* Always use the phrase 'AMP Cache" instead of 'Cache' as it might be confusing with CDNs etc.
* Maybe make this more explicit: "Your Core Web Vitals might suffer if your AMP page performs really great when served from an AMP Cache, but are really slow on the origin." (please rephrase to English)

Reply:
updated wording - wdyt